.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,6 +4,6 @@ about: Suggest a new feature for this project
 labels: 'enhancement'
 ---
 
-Features that impact the whole project (e.g. Adding a new OpenJDK distribution) are made over at the [TSC repository](https://github.com/AdoptOpenJDK/TSC/).
+Features that impact the whole project (e.g. Adding a new OpenJDK distribution) are proposed and discussed in the [TSC repository](https://github.com/AdoptOpenJDK/TSC/).
```

Comment:
It's no longer the TSC repo but the [main Adoptium repository](https://github.com/adoptium/adoptium/)

Reply:
I have fixed the link